src/main/java/org/embulk/util/csv/InvalidCsvColumnsException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2014 The Embulk project
```

Comment:
2014 Intentionally?

Reply:
Ah yes, it actually came from `org.embulk.standards.CsvTokenizer.InvalidFormatException` in 2014.